src/com/intellij/idea/plugin/hybris/tools/logging/CxLoggerAccess.kt

```diff
@@ -120,7 +119,7 @@ class CxLoggerAccess(private val project: Project, private val coroutineScope: C
 
     private fun updateCache(loggers: Map<String, CxLoggerModel>?) {
         coroutineScope.launch {
-            putUserData(KEY_LOGGERS_STATE, loggers)
+            putUserData(KEY_LOGGERS_STATE, CxLoggersStorage(loggers?.toMutableMap() ?: mutableMapOf()))
```

Comment:
let's ensure that cached value is not modifiable during re-evaluation.

instead, it is preferable to replace cache with the completely new loggers